engines/suppliers/Gemfile.lock

```diff
@@ -2,68 +2,93 @@ PATH
   remote: .
   specs:
     suppliers (0.0.1)
-      rails (~> 4.1.7)
+      rails (~> 4.2.5)
```

Comment:
Except if an engine needs another local engine, or a private gem, or a gem downloaded from a specific git commit at some repo. In that case there is no way to tell the outside world this dependency, so a Gemfile is really useful. No the Gemfile.lock though.